README.md

```diff
@@ -41,5 +40,12 @@ Then layer `boundaries` can be served with:
 
 `fields` is optional, and when absent only geometries are encoded in the vector tile.
 
+## Preview 
+
+When the source is a tm2 file, the root endpoint will display a built-in viewer with `mapbox-gl-js`.
```

Comment:
You also have a viewer when pointing to a mbtiles right?

Reply:
Thinking out loud (for another PR): when using a direct connexion to postgis, the index.html could display a form asking for the layer name to display, before we can implement our mechanism for discovering postgis layer.

Reply:
> You also have a viewer when pointing to a mbtiles right?

Fixed !

Reply:
> Thinking out loud (for another PR): when using a direct connexion to postgis, the index.html could display a form asking for the layer name to display, before we can implement our mechanism for discovering postgis layer.

For that we also need to apply a ramdom style dynamically.

Reply:
I think I'm missing something here: we can't apply - for instance - the style.json in Oslandia/postile-openmaptiles?